src/emulator/auth/index.ts

```diff
@@ -13,6 +13,8 @@ export interface AuthEmulatorArgs {
   projectId: string;
   port?: number;
   host?: string;
+  singleProjectModeWarning?: boolean;
```

Comment:
Can we have an `enum` instead of two booleans? Like `ENFORCED`, `WARN`, and `DISABLED`?

Reply:
Added